packages/esm-patient-common-lib/src/types/workspace.ts

```diff
@@ -1,14 +1,21 @@
 /* The possible states a workspace window can be opened in. */
 export type WorkspaceWindowState = 'maximized' | 'hidden' | 'normal';
 
+export interface CloseWorkspaceOptions {
+  workspaceTitle?: string;
+  ignoreChanges?: boolean;
+  onWorkspaceClose?: () => void;
+  confirmBeforeClosing?: boolean;
```

Comment:
These will need docstrings. `confirmBeforeClosing` appears not to be used. Let's get rid of that. And `workspaceTitle` does something strange—why would the caller of `closeWorkspace` know the title, but the title not have been provided by the workspace in registration? Maybe there should be some fallback behavior if no title is defined, but this seems strange to me.